.github/workflows/deploy.yml

```
@@ -164,29 +164,30 @@ jobs:
 
   build-web:
     runs-on: ubuntu-latest
-    defaults:
-      run:
-        shell: bash
     steps:
       - name: Checkout
         uses: actions/checkout@v2
         with:
           ref: master
 
-      - name: Setup Hugo
-        uses: peaceiris/actions-hugo@v2
+      - name: Setup Ruby
+        uses: actions/setup-ruby@v1
         with:
-          hugo-version: '0.79.1'
+          ruby-version: ${{ env.RUBY_VERSION }}
+
+      - name: Install dependencies
+        run: bundle install --jobs=3 --retry=3
+        working-directory: ./web
```

Rather than this, shouldn't you use `setup-ruby` with a working directory and use `bundler-cache: true`?

```suggestion
          ruby-version: ${{ env.RUBY_VERSION }}
          bundler-cache: true
          working-directory: ./web
```

---

Sure let me try that, makes sense.